index.js

```diff
@@ -1034,6 +1039,69 @@ const createGoBom = async (includeBomSerialNumber, path, options, callback) => {
       },
       callback
     );
+    return;
+  }
+
+  // If USE_GOSUM is false, generate BOM components using go.mod.
+  gosumMap = {};
+  if (gosumFiles.length) {
+    for (let i in gosumFiles) {
+      const f = gosumFiles[i];
+      if (DEBUG_MODE) {
+        console.log(`Parsing ${f}`);
+      }
+      data = fs.readFileSync(f, { encoding: "utf-8" });
+      checkSums = data.split("\n")
+      for (let d in checkSums) {
+        const l = checkSums[d];
+        if (l.length > 0) {
```

Comment:
Nice one! Not a biggie that this might not be effective if the line is empty but with a space character.

Reply:
Yeah, in this case the go.sum should always be generated, and the /linendings are likely dependent on the machine the go.sum was generated on. In theory a go.sum shouldn't ever container an empty line with added whitespace